src/index.html

```diff
@@ -22,6 +22,369 @@
   </head>
 
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a
+        class="header__logo"
+        href="#"
+      >
+        <img
+          src="images/logo.png"
+          alt="logo"
+        />
+      </a>
+      <nav class="header__nav">
+        <ul class="header__nav-list">
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link is-active"
+              href="#apple"
+            >
+              Apple
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#samsung"
+            >
+              Samsung
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#smartphones"
+            >
+              Smartphones
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#laptops&computers"
+              data-qa="nav-hover"
+            >
+              Laptops & Computers
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#gadgets"
+            >
+              Gadgets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#tablets"
+            >
+              Tablets
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#photo"
+            >
+              Photo
+            </a>
+          </li>
+
+          <li class="header__nav-item">
+            <a
+              class="header__nav-link"
+              href="#video"
+            >
+              Video
+            </a>
+          </li>
+        </ul>
+      </nav>
+    </header>
+    <main class="catalog">
+      <div
+        class="card"
+        data-qa="card"
+      >
+        <img
+          src="./images/imac.jpeg"
+          alt="card__image"
+          class="card__img"
+        />
+        <div class="card__description">
+          <h4 class="card__title">
+            APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)
+          </h4>
+          <div class="card__code">Product code: 195434</div>
```

Comment:
You need to use semantic tag for text, fix it everywhere
```suggestion
          <p class="card__code">Product code: 195434</p>
```